docs/core/deploying/single-file.md

```diff
@@ -139,18 +139,26 @@ These properties have the following functions:
 Publish a single file application using the [dotnet publish](../tools/dotnet-publish.md) command. When you publish your app, set the following properties:
 
 - Publish for a specific runtime: `-r win-x64`
-- Publish as a single-file: `-p:PublishSingleFile=true`
+- Publish as a single-file: `<PublishSingleFile>true</PublishSingleFile>`
 
 The following example publishes an app for Windows as a self-contained single file application.
 
-```dotnetcli
-dotnet publish -r win-x64 -p:PublishSingleFile=true --self-contained true
```

Comment:
Maybe we should list both the XML project settings and the command line options, and then explain the advantages to setting the XML project settings?

My concern here is that new developers could be confused that they can't set these options on the command line (especially the `RuntimeIdentifier`).